src/nni_manager/rest_server/restHandler.ts

```diff
@@ -262,6 +264,28 @@ class NNIRestHandler {
         });
     }
 
+    private getMetricDataByRange(router: Router): void {
+        router.get('/metric-data-range/:min_seq_id/:max_seq_id', async (req: Request, res: Response) => {
```

Comment:
@chicm-ms What's the RESTful way to request a range of resources?

Reply:
Using this is ok, or use query string,  such as /metric-data-range?min_seq_id=1&max_seq_id=2